src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,12 +10,21 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
+    public static final String EXAMPLE = "123, some street, some unit number, some postal code";
```

Comment:
`some street` is barely acceptable as an example street name, but `some unit number` is not. :-p